### PR TITLE
edit html

### PR DESCRIPTION
**Pull request by Trollering**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,5 +6,9 @@
 		<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	</head>
 	<body>
+		<h1>HTML</h1>
```
Comment (JayoonKoo) [inline index.html:9]: Change Tag from h1 to h2

File: index.html [hunk 2]
```diff
@@ -6,5 +6,9 @@
...
+		<p>
+		The HyperText Markup Language, or HTML is the standard markup language for documents designed to be displayed in a web browser.
+		</p>
```
Comment (JayoonKoo) [inline index.html:12]: could you add anything using ul tag?

**Review by JayoonKoo — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (Trollering) [reply index.html]: Ok.

File: index.html (hunk 2 above)
Comment (Trollering) [reply index.html:12]: Ok.

**Review by JayoonKoo — APPROVED** [review]

**Pull request merged**